lib/rbUtil.js

```diff
@@ -221,7 +221,7 @@ rbUtil.cloneRequest = function(req) {
         method: req.method || 'get',
         headers: req.headers || {},
         query: req.query || {},
-        body: req.body || null,
+        body: req.body || (req.body === '' ? '' : null),
```

Comment:
Nit: we actually want to keep whatever is supplied, if it's defined. So, perhaps it would be better to phrase this as:

```js
body: req.body !== undefined ? req.body : null,
```